tests/Place/PlaceCommandHandlerTest.php

```diff
@@ -14,20 +14,15 @@
 use CultuurNet\UDB3\CalendarType;
 use CultuurNet\UDB3\Description;
 use CultuurNet\UDB3\Event\EventType;
-use CultuurNet\UDB3\Label;
```

Comment:
Some extra imports to remove
```
use CultuurNet\UDB3\Label\LabelServiceInterface;
use CultuurNet\UDB3\Label\ReadModels\JSON\Repository\Entity;
use CultuurNet\UDB3\Label\ReadModels\JSON\Repository\ReadRepositoryInterface;
use CultuurNet\UDB3\Label\ValueObjects\Privacy;
use CultuurNet\UDB3\Label\ValueObjects\Visibility;

use ValueObjects\Identity\UUID;

use ValueObjects\StringLiteral\StringLiteral;
```

Reply:
The first 5 are already removed in a later commit, the last 2 are removed here: d1009c8